test/BridgeExecutorBase.spec.ts

```diff
@@ -399,6 +400,46 @@ describe('BridgeExecutorBase', async function () {
       expect((await bridgeExecutor.getActionsSetById(0)).executed).to.be.equal(true);
     });
 
+    it('Queue and fail to execute an actions set with insufficient value', async () => {
```

Comment:
Following the style of the tests, the title would be "Tries to queue.... value (revert expected)'

Reply:
Sure, will fix.

Reply:
I fixed this but I don't like this convention long-term. I think we should separate revert cases from success cases in separate sections instead. This naming convention can get a little confusing if the test contains multiple transactions (which, admittedly isn't often, but still).

Reply:
Yeah, thats a good point actually